locallibrary/catalog/views.py

```diff
@@ -25,3 +38,22 @@ def index(request):
     # Render the HTML template index.html with the data in the context variable
     return render(request, 'index.html', context=context)
 
+class BookListView(generic.ListView):
+    model = Book
+    paginate_by = ITEMS_PER_PAGE
+
+    def get_queryset(self):
+        # Su dung select_related de lay thong tin cua author cung luc khi query book
+        return Book.objects.select_related('author').all()
```

Comment:
phân biệt `select_related` và `prefetch_related`

Reply:
select_related sử dụng cho  quan hệ one-to-one hoặc many-to-one
prefetch_related sử dụng cho quan hệ many-to-many hoặc reverse many-to-one ạ